// program to demonstrate Throws
class AR
{
	public void div(int a,int b)throws ArithmeticException
	{
		int res =a/b;
		System.out.println("res="+res);
	}
}
public class P46 {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		System.out.println("main starts");
		try
		{
			AR a=new AR();
			a.div(4,0);
		}
		catch(ArithmeticException e)
		{
			System.out.println("Div by zero "+e);
		}
		System.out.println("main ends");
	}

}

*****************************output*****************************************
main starts
Div by zero java.lang.ArithmeticException: / by zero
main ends